source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/point/NcaCustomCalculationPoint.java

```diff
@@ -24,6 +26,7 @@ public class NcaCustomCalculationPoint extends CustomCalculationPoint {
   private static final long serialVersionUID = 1L;
 
   private Double roadLocalFractionNO2;
+  private List<EntityReference> entityReferences;
```

Comment:
Generally we already initialize collections in `domain.v2`, in the case of lists usually with an `ArrayList`.